raml2swagger-lib/src/main/java/org/macrulez/utils/raml2swagger/RAMLtoSwagger.java

```diff
@@ -635,7 +641,23 @@ private JSONObject getResponseInfo(Map.Entry<ActionType, Action> action) throws
                         fields.put(SCHEMA_PARAM_KEY, jsonObj);
                     }
                 }
+
+                if (example != null && example.length() > 0) {
+                    // If example is JSON, let's format it
+                    if (MIMETYPE_JSON.equals(me.getKey())) {
+                        JSONObject jsonObj = new JSONObject(example);
```

Comment:
I would use `new JSONTokener(example).nextValue()` to accommodate JSON arrays too.

Reply:
@sslb, could you look into that part of implementation?